### PR TITLE
move some methods to utils.py, to prepare STRIPS learning refactor

### PR DESCRIPTION
**Pull request by ronuchit**

### CODE REVIEW

File: src/nsrt_learning/option_learning.py [hunk 1]
```diff
@@ -7,13 +7,14 @@
 from typing import Dict, List, Sequence, Set, Tuple
 
 import numpy as np
+from gym.spaces import Box
```
Comment (NishanthJKumar) [inline src/nsrt_learning/option_learning.py:10]: was this intentional? Why is this the only change to this file?

File: tests/test_predicate_search_score_functions.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 import numpy as np
 import pytest
+from gym.spaces import Box
```
Comment (NishanthJKumar) [inline tests/test_predicate_search_score_functions.py:7]: again, don't we want to import Box from `structs.py`?

**Review by NishanthJKumar — APPROVED** [review]
LGTM - aside from the changes that import `Box` from `gym` instead of `structs`. Feel free to merge if that was intentional, but do fix if not!

**Comment by ronuchit** [comment]
it's intentional; Box is not something we define in structs.py, it's from openai gym. we had incorrectly imported it from structs in some places

**Pull request merged**